UI/UnoSQLiteOneDriveInvoiceSample/Demo/Demo/Demo.Shared/Helpers/BaseNotifyClass.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
+
+namespace Demo.Helpers
+{
+    public class BaseNotifyClass : INotifyPropertyChanged
```

Comment:
often a class like this will be called something like `ViewModel` or `ViewModelBase` :)